Lib/test/test_atexit.py

```diff
@@ -46,6 +48,43 @@ def test_atexit_instances(self):
         self.assertEqual(res.out.decode().splitlines(), ["atexit2", "atexit1"])
         self.assertFalse(res.err)
 
+    def test_multiprocessing(self):
```

Comment:
It is a little awkward to have logic specific to multiprocessing start methods on different platforms in test_atexit.py.  I think this test would be better off within something like `_test_multprocessing.py` which is turned into multiple separate test suites per start method based upon what the platform supports.

Reply:
I moved the test to `_test_multiprocessing.py`